README.md

```diff
@@ -541,6 +543,11 @@ These settings can be changed at any time.
   Bottlerocket enables the `json-file`, `awslogs`, and `none` drivers by default.
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.metadata-service-rps`: The steady state rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.metadata-service-burst`: The burst rate limit of the throttling configurations set for the task metadata service
```

Comment:
For the `metadata-service-rps` and `metadata-service-burst` settings, I think we should add a small snippet of documentation commenting on the correlation between these two settings in the Bottlerocket API and the comma separated string for `ECS_TASK_METADATA_RPS_LIMIT` in the agent.

Even something like the following would suffice:

> "these two settings will be combined to generate the `ECS_TASK_METADATA_RPS_LIMIT` ECS agent setting"

I _don't_ think it'll be immediate clear to users that these two values are related to the `ECS_TASK_METADATA_RPS_LIMIT` agent setting. [The ECS agent documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html) _is_ clear what this setting does, but a quick scan of setting names doesn't mention anything about "burst" so I see this as a likely place of confusion for users without abit of extra documentation.

Reply:
In https://github.com/bottlerocket-os/bottlerocket/pull/2527#discussion_r1007677221, it was decided to drop the `rate` part of the setting (which was similar to the suggestion you make with `limit`) to be consistent with other existing APIs.

I do think that I can make this setting a bit more "clear" by adding the prefix `task-`:

- `task-metadata-service-rps`
- `task-metadata-service-burst`

But if folks don't feel strongly about this, I can keep what I have :sweat_smile:

Reply:
I would prefer `task-metadata-service-*`, but I don't feel too strongly about it.